inc/rest-api.php

```diff
@@ -165,6 +165,24 @@ function rest_user_search( \WP_REST_Request $request ) {
 	return rest_ensure_response( $data );
 }
 
+/**
+ * Filter REST API dispath request to skip ES integration. Disables VIP
+ * Enterprise Search for Profiles to support Byline Manager's byline and user
+ * selectors in Gutenberg.
+ *
+ * @param mixed            $result Response to replace the requested version with.
+ * @param \WP_REST_Server  $server Server instance.
+ * @param \WP_REST_Request $request Request used to generate the response.
+ * @return mixed Filtered response.
+ */
+function filter_rest_pre_dispatch( $result, $server, $request ) {
```

Comment:
Interesting that this is what is breaking the ability to search for this post type in the admin. I would assume that if EP was not activated, then things would just work without this filter. Is that not the case?

Reply:
@joemcgill Yes, confirmed. If ES is not activated, then the `/byline-manager/v1/authors` endpoint just works and so does the byline selector. This changes support the byline selector when ES is enabled.